src/script/interpreter.cpp

```diff
@@ -1259,33 +1259,33 @@ class CTransactionSignatureSerializer
 };
 
 template <class T>
-uint256 GetPrevoutHash(const T& txTo)
+uint256 GetPrevoutSHA256(const T& txTo)
```

Comment:
Might as well rename these to plural prevout**s** and sequence**s**: `GetPrevoutsSHA256` and `GetSequencesSHA256`.

Reply:
done.